Makefile

```diff
@@ -73,6 +73,7 @@ TELE_BUILD_OPTIONS := --name=$(NAME) \
 
 BUILD_DIR := build
 BINARIES_DIR := bin
+TARBALL := build/application.tar
```

Comment:
I think this rename will break robotest unless you update the tarball name in its scripts too,

Reply:
This is the name for the application tarball. Robotest is using cluster image tarball which is still `build/installer.tar`.